workshop.wwwapi/Endpoints/SurgeryEndpoint.cs

```diff
@@ -10,24 +11,96 @@ public static void ConfigurePatientEndpoint(this WebApplication app)
         {
             var surgeryGroup = app.MapGroup("surgery");
 
-            surgeryGroup.MapGet("/patients", GetPatients);
             surgeryGroup.MapGet("/doctors", GetDoctors);
-            surgeryGroup.MapGet("/appointmentsbydoctor/{id}", GetAppointmentsByDoctor);
+            surgeryGroup.MapGet("/doctors/{id}", GetDoctorById);
+            surgeryGroup.MapPost("/doctors/{id}", AddDoctor);
```

Comment:
add doctor -> why do you pass the ID? the ID is going to be created server-side...